contents/page-sections/hero-headers/with-background-image/index.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Center,
+  Container,
+  Heading,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const HeroWithBackgroundImage: FC = () => {
+  return (
+    <Container
+      bgImage="url(https://images.unsplash.com/photo-1573164713988-8665fc963095?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=980&q=80)"
+      bgSize="cover"
+      bgPosition="center"
+      minH="fit-content"
+      h="full"
+    >
+      <Box position="absolute" inset={0} zIndex="0" bg="blackAlpha.700" />
+      <VStack as={Center} px="xl" h="full" zIndex="1" py="xl" gap="lg">
```

Comment:
If we specify `px` and `py` as the same thing, why not just use `p`?
And I think you should change the padding only for `md` and `sm`.

```suggestion
      <VStack as={Center} h="full" zIndex="1" p={{base: "lg", md: "md"}} gap="lg">
```

Reply:
Fixed.